README.md

```diff
@@ -152,7 +152,7 @@ misc arguments:
 If you want to run AutoSploit on a macOS system, AutoSploit is compatible with macOS, however, you have to be inside a virtual environment for it to run successfully. To do this, do the following;
 
 ```bash
-sudo -s << '_EOF' 
+sudo -s << '_EOF'
```

Comment:
Same as a above, why are you wanting to use single quotes in the commands?

Reply:
I didn't make any actual change to this line. [Current master brand README.md here](https://github.com/NullArray/AutoSploit/blob/master/README.md) - It apparently had a space after the `'_EOF'` line and my editor removed it automatically but the quotes were already there. I'm not sure why the author would've put that or why they put an underscore before `EOF`.

Reply:
Oh okay, I mean it doesn’t really matter in the command was just curious, I put the underscore just because that’s what I’ve always don’t in these kind of commands (idk why tbh) lol

Reply:
No worries :) I stick with my habits too, `_EOF` is a lot less likely to be in a heredoc than `EOF` in your defense if you're going to pick one to always use.